index.html

```diff
@@ -372,5 +402,177 @@ <h2>Buttons</h2>
+                <option value="2">Item 2</option>
+                <option value="3">Item 3</option>
+            </select>
+
+            <select class="mi-select" disabled>
+                <option value="" disabled selected>Select an item</option>
+            </select>
+        </div>
+    </section>
+
+    <hr>
+
+    <section class="mi-inputs">
+        <header>
+            <h2>Input</h2>
+        </header>
+
+        <div class="row">
+            <label class="mi-label">
+                Text
+                <input type="text" value="value" class="mi-input">
+            </label>
+
+            <label class="mi-label">
+                Text disabled
+                <input type="text" value="value" class="mi-input" disabled>
+            </label>
+        </div>
+
+        <div class="row">
+            <label class="mi-label">
+                Number
+                <input type="number" value="1" class="mi-input">
+            </label>
+
+            <label class="mi-label">
+                Number disabled
+                <input type="number" value="1" disabled class="mi-input">
+            </label>
+        </div>
+
+
+
+        <div class="row">
+            <div class="column">
+                Radio
+                <div>
+                    <label class="mi-label mi-label--right">
+                        <input type="radio" name="group1" value="1" checked class="mi-input">
+                        Choice 1
+                    </label>
+                    <label class="mi-label mi-label--right">
+                        <input type="radio" name="group1" value="2" class="mi-input">
+                        Choice 2
+                    </label>
+                    <label class="mi-label mi-label--right">
+                        <input type="radio" name="group1" value="3" class="mi-input">
+                        Choice 3
+                    </label>
+                </div>
+            </div>
+
+            <div class="column">
+                Radio disabled
+                <div>
+                    <label class="mi-label mi-label--right">
+                        <input type="radio" name="group2" value="1" checked disabled class="mi-input">
+                        Choice 1
+                    </label>
+                    <label class="mi-label mi-label--right">
+                        <input type="radio" name="group2" value="2" disabled class="mi-input">
+                        Choice 2
+                    </label>
+                    <label class="mi-label mi-label--right">
+                        <input type="radio" name="group2" value="3" disabled class="mi-input">
+                        Choice 3
+                    </label>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="row">
+            <div class="column">
+                Checkbox
+                <div>
+                    <label class="mi-label mi-label--right">
```

Comment:
I think we can manage to avoid this modifier class. I tried to remove it, and it can work for radio and checkboxes. Toggle however needs some more work.

![image](https://user-images.githubusercontent.com/50314165/121479835-bad48a00-c9ca-11eb-93fe-f6b6187e5f4c.png)